docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '2'
 services:
     node:
         # Once Wikimedia has node12 in production docker-registry, use that.
```

Comment:
This comment should probably be `node14` now?

Reply:
Okay, so maybe we should just remove the comment? If it’s just a development image, it shouldn’t be so important which Node image we use, right?